src/map/main/MapLiveRoutes.js

```diff
@@ -16,6 +16,15 @@ const MapLiveRoutes = () => {
 
   const history = useSelector((state) => state.session.history);
 
+  const reportWidth = useSelector((state) => {
+    return state.session.user.attributes['web.routeWidth'] || state.session.server.attributes['web.routeWidth'] || 2;
+    
+  });
+
+  const reportOpacity = useSelector((state) => {
```

Comment:
Why are you not using `usePreference` for all of these?

Reply:
I'll change, hadn't found it. Assume `useAttributePreference` ?

Reply:
Yes, I think one of those.